### ISSUE TITLE
Installation problems 

### ISSUE THREAD

**Issue by NoobRebounder** [hannorein/rebound#558]
Hi, 

I'm a complete noob and using a mac running python 2.7.10. I've tried using the simple instructions on the website but this is what I get.

   pip instal rebound
             ^
SyntaxError: invalid syntax
>>> pip install rebound
  File "<stdin>", line 1
    pip install rebound

The instructions using a virtual environment on youtube don't work either.

Any help is greatly appreciated.

**Comment by Findus23** [comment]
Hi,

pip is a command-line program, so you need to run it in the shell of your choice instead of in python.
How exactly you do this depends a lot on your computer setup and might be out of scope to explain in the rebound docs.

**Comment by NoobRebounder** [comment]
Thanks for the quick reply. 

I'm very new to this, but used to use linux long ago. From what I can tell linux has built in package managers but it appears that Mac requires a package manager like Homebrew or something similar.

Am I kind of on the right track?

Thanks again.

**Comment by hannorein** [comment]
No. Python+pip come preinstalled with OSX. Simply typing `pip install rebound` into a command line as @Findus23 said should work.

**Comment by NoobRebounder** [comment]
Thanks for the fast reply, but unfortunately that doesn't work. This is what I get when I open a terminal and use that command.

users-MacBook-Air-2:~ user$ pip install rebound
-bash: pip: command not found

**Comment by Rmelikyan** [comment]
Try `pip3 install rebound`

Then to use, run with `python3` instead of `python` which will default to version 2.7.

You may want to look into virtual environments as well when you are testing installations like this. This article helped me through a lot of these issues: https://docs.python-guide.org/dev/virtualenvs/#basic-usage

**Comment by NoobRebounder** [comment]
Unfortunately it says pip3: command not found

I'm thinking of reformatting the computer and starting again. I do appreciate the help though.

**Comment by hannorein** [comment]
This sounds really strange. Starting with a fresh OSX installation should restore everything. Other wise you might want to look into some other distribution like anaconda.  In the end, this is not so much a REBOUND but python issue so I'll close the issue for now.

**Comment by Rmelikyan** [comment]
Good luck!

This seems to me like a programming environment problem rather than a REBOUND problem. I'd suggest finding a guide in getting python set up for use on your computer. Consider Anaconda for a very inclusive package which will immediately give you access to jupyter notebooks (assuming this is how you want to develop your code with REBOUND).

Maybe close this issue for now and reopen if you are still having problems after setting up your python environment.

Getting started is always the frustrating part... Once this is sorted out I hope you will find REBOUND easy to use and very useful for your work!
